src/libratbag.h

```diff
@@ -1300,6 +1322,139 @@ enum ratbag_button_action_special
 ratbag_button_get_special(struct ratbag_button *button);
 
 /**
+ * @ingroup led
+ *
+ * LED mods.
+ */
+enum ratbag_led_mode {
+	RATBAG_LED_OFF = 0,
+	RATBAG_LED_ON = 1,
+	RATBAG_LED_CYCLE = 03,
+	RATBAG_LED_BREATH = 10
+};
+/**
```

Comment:
let's make this a normal sequential enum and not leak implementation detail. Means we need to convert types somewhere, but that's a mere switch statement - which we'd need in every driver that doesn't use hidpp20 constants anyway